include/seqan/modifier/modifier_iterator.h

```diff
@@ -538,11 +530,20 @@ position(ModifiedIterator<THost, TSpec> const & me)
     return position(host(me));
 }
 
+// rooted overload
+template <typename TContainer1, typename TIterator, typename TSpec1, typename TSpec2, typename TContainer2>
+inline typename Position<ModifiedIterator<Iter<TContainer1, AdaptorIterator<TIterator, TSpec1> >, TSpec2> const>::Type
+position(ModifiedIterator<Iter<TContainer1, AdaptorIterator<TIterator, TSpec1> >, TSpec2> const & me,
+        TContainer2 const &)
```

Comment:
Unfortunately not, and our rooted iterator doesn't even fulfill our rooted iterator concept :S
Even if it this were fixed it is not possiblr to do globally, because the modified iterator of a rooted iterator is a specialization of the modified iterator and not a specialization of the rooted iterator.